analysis/analysis.py

```diff
@@ -86,6 +133,20 @@ def test_compute_total_view_time(self):
         total_view_time = compute_total_view_time(self.default_student_data)
         self.assertEqual(total_view_time, 0)
 
+    def test_common_response(self):
+        """
+        Tests to make sure the function runs properly by checking against known data sets.
+        :return: None
```

Comment:
for a short testing function like this, a one-line docstring is fine -- you don't need to specify the return value here